src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -181,6 +181,16 @@
     <xsd:anyAttribute namespace="##any" processContents="lax" />
   </xsd:complexType>
 
+  <xsd:complexType name="deprecated">
+    <xsd:simpleContent>
+      <xsd:extension base="xsd:string">
+        <xsd:attribute name="message" type="xsd:string" />
```

Comment:
should be removed, in favor of the content

Reply:
Done